orderbook/openapi.yml

```diff
@@ -447,7 +447,7 @@ components:
     OrderStatus:
       description: The current order status
       type: string
-      enum: [open, fulfilled, cancelled, expired]
+      enum: [signaturePending, open, fulfilled, cancelled, expired]
```

Comment:
@anxolin @W3stside @alfetopito 

Just pinging you guys so you are aware of the new order status (for an order that was accepted by the orderbook and is waiting for the on-chain presignature).

Reply:
Thanks for letting us know.
I've created one task for the explorer https://github.com/gnosis/gp-ui/issues/628

I believe Anxo is taking care of cowswap changes.